src/town_cmd.cpp

```diff
@@ -1113,6 +1113,7 @@ static bool GrowTownWithBridge(const Town *t, const TileIndex tile, const DiagDi
 
 	/* We are in the right direction */
 	uint8 bridge_length = 0;      // This value stores the length of the possible bridge
+	TileIndex tile_past_bridge = tile; // We check the tile beyond the other end of the bridge to avoid building a dead-end bridge
```

Comment:
Initialization to tile is superfluous, as it is set below anyway.

Reply:
Right. Thanks.